pkg/resources/database.go

```diff
@@ -62,6 +62,32 @@ var databaseSchema = map[string]*schema.Schema{
 		ForceNew:      true,
 		ConflictsWith: []string{"from_share", "from_database"},
 	},
+	// TODO: Add accounts for replication (it will promote local database to serve as a primary database for replication).
```

Comment:
Those values should be in the standard database, so either here or in the new resource. I'll bring it up tomorrow.

Reply:
any news?

Reply:
I'll remove this to-do in the next PR, but essentially it will be possible to enable replication and failover in the new "standard" database resource (but maybe in different format than it was previously done).

Reply:
Okay, it's good for now, we can do the split later.